### ISSUE TITLE
CMake Configure should fail if SWIG isn't found

### ISSUE THREAD

**Issue by player1537** [SENSEI-insitu/SENSEI#84]
I was trying to compile v4.0.0 and although the CMake Configure step was successful, the build fails near the end when trying to compile `svtk.i`.

I believe the simplest fix is to simply set SWIG as a `REQUIRED` package within the `CMake/python.cmake`. A trickier fix would be to disable Python support as a whole if SWIG isn't found.

The symptom is a `No Such File or Directory` where the previous line includes `SWIG_EXECUTABLE-NOTFOUND`:

```console
$ cmake --build ./build
[. . .]
cd /home/visuser/source/tanner/playground/var/sensei-4.0.0/utils/pysvtk && /opt/ohpc/pub/utils/cmake/3.21.3/bin/cmake -E env SWIG_LIB= SWIG_EXECUTABLE-NOTFOUND [. . .]
```

**Comment by kwryankrattiger** [comment]
The swig setup in `CMake/python.cmake` is a bit strange. It seems like there was an older version that may have hand-rolled swig stuff and was changed to use the `UseSWIG` module.

**Comment by kwryankrattiger** [comment]
Fixed in #85

**Comment by burlen** [comment]
yes, that is the case. sensei 4 adopted cmake's swig driver
